src/pretix/presale/templates/pretixpresale/organizers/index.html

```diff
@@ -84,7 +84,11 @@ <h3>{% trans "Upcoming events" %}</h3>
                             {% if e.best_availability_is_low %}
                                 <span class="label label-success-warning">{% trans "Few tickets left" %}</span>
                             {% else %}
-                                <span class="label label-success">{% trans "Book now" %}</span>
+                                {% if event.event.has_paid_item %}
```

Comment:
Müsste das hier `e` sein? Siehe zwei Zeilen weiter oben `e.best_availability_is_low`
```suggestion
                                {% if e.has_paid_item %}
```